runtime/centrifuge/src/lib.rs

```diff
@@ -1180,6 +1277,37 @@ impl pallet_collator_selection::Config for Runtime {
 	type WeightInfo = weights::pallet_collator_selection::WeightInfo<Self>;
 }
 
+/// Xcm Weigher shared between multiple Xcm-related configs.
+pub type XcmWeigher = xcm_builder::FixedWeightBounds<UnitWeightCost, RuntimeCall, MaxInstructions>;
+
+parameter_types! {
+	// TODO(cdamian): is this correct?
+	// 1 DOT should be enough to cover for fees opening/accepting hrmp channels
+	pub MaxHrmpRelayFee: MultiAsset = (MultiLocation::parent(), 1_000_000_000_000u128).into();
```

Comment:
Are we OK with this value?

Reply:
AFAICT this is way too high: DOT has [10 decimals](https://wiki.polkadot.network/docs/learn-DOT#polkadot), so this value is actually 100 DOT.

Reply:
Should we go with 1 DOT or less?